Docs/source/developers/amrex_basics.rst

```diff
@@ -0,0 +1,26 @@
+AMReX basics (excessively basic)
+================================
+
+WarpX is build on the Adaptive Mesh Refinement (AMR) library `AMReX <https://github.com/AMReX-Codes/amrex>`__. This section provides a very sporadic description of the main AMReX classes and concepts relevant for WarpX, that can serve as a reminder. Please read the AMReX basics `doc page <https://amrex-codes.github.io/amrex/docs_html/Basics.html>`__, of which this section is largely inspired.
```

Comment:
Is there a difference between one and two underscores in links?

Reply:
There seems to be, but that's still not very clear to me 
https://stackoverflow.com/questions/27420317/restructured-text-rst-http-links-underscore-vs-use